packages/bpk-component-radio/src/__snapshots__/BpkRadio-test.js.snap

```diff
@@ -108,6 +108,7 @@ exports[`BpkRadio should render correctly with checked attribute 1`] = `
     <input
       aria-invalid="false"
       aria-label="Direct"
+      checked=""
```

Comment:
Might be worth checking why this has changed in this PR? Possibly revert it as its not related and expected with this change